packages/paste-core/components/spinner/stories/index.stories.tsx

```diff
@@ -1,12 +1,26 @@
 import * as React from 'react';
 import {storiesOf} from '@storybook/react';
-import {withKnobs, text, number} from '@storybook/addon-knobs';
+import {withKnobs, text, select} from '@storybook/addon-knobs';
+import {DefaultTheme, ThemeShape} from '@twilio-paste/theme-tokens';
 import {Spinner} from '../src';
 
+const IconColorOptions = Object.keys(DefaultTheme.textColors);
+const SizeOptions = Object.keys(DefaultTheme.iconSizes);
+
 storiesOf('Components|Spinner', module)
   .addDecorator(withKnobs)
   .add('Default', () => {
+    const iconColorValue = select('iconColor', IconColorOptions, 'currentColor') as keyof ThemeShape['textColors'];
```

Comment:
You can use the types package here, can't you?

```
import {TextColor, IconSize} from '@twilio-paste/types'

...

const iconColorValue = select('iconColor', IconColorOptions, 'currentColor') as TextColor;
```

Reply:
Good point, fixed.